### DISCUSSION TITLE
converting world's save form 1.7.1.0 tro 1.8.0.1

### DISCUSSION THREAD

**Discussion by soniqexn** [Papela/Nitrox-Cracked-Mod#12] [General]
is there a way to import 1.7.1.0 saves to 1.8.0.1 ?
the import button reset everything from my preview world

**Comment by Papela** [comment]
No, there is no way to convert worlds right now and it doesn't look like there are any planned for the near future. I recommend creating a new world to be able to play on v1.8.0.2.
